### ISSUE TITLE
USE_OCVSIFT is not defined for main_ComputeFeatures_OpenCV

### ISSUE THREAD

**Issue by tomas789** [openMVG/openMVG#1095]
I cannot build openMVG with SIFT from OpenCV. Everything builds correctly but OpenCV SIFT is not included in `openMVG_main_ComputeFeatures_OpenCV` although I specified it in `cmake` call
```
cmake -DOpenMVG_USE_OPENCV=ON -DOpenMVG_USE_OCVSIFT=ON ../src
```

Problem is likely caused by `openMVG/src/software/SfM/CMakeLists.txt` on line 341. This
```
target_compile_definitions(openMVG_main_ComputeFeatures PRIVATE USE_OCVSIFT)
```
should be probably replaced by this
```
target_compile_definitions(openMVG_main_ComputeFeatures_OpenCV PRIVATE USE_OCVSIFT=1)
```

With this change, I get expected result
```
➜  build git:(master) ✗ ./Darwin-x86_64-Release/openMVG_main_ComputeFeatures_OpenCV                                     
Usage: ./Darwin-x86_64-Release/openMVG_main_ComputeFeatures_OpenCV
[-i|--input_file]: a SfM_Data file 
[-o|--outdir] path 

[Optional]
[-f|--force] Force to recompute data
[-m|--describerMethod]
  (method to use to describe an image):
   AKAZE_OPENCV (default),
   SIFT_OPENCV: SIFT FROM OPENCV

Invalid command line parameter.
```

**Comment by pmoulon** [comment]
Thanks for the report, it's a mistake ;-)
Please feel free to make a Pull Request in order to fix this.

BTW, I would like to advise you to use the internal openMVG SIFT. They are giving better results that OpenCV sift method.

**Comment by pmoulon** [comment]
Fixed by #1098
